src/styles/fonts.scss

```diff
@@ -0,0 +1,6 @@
+:root {
+  font-family: Roboto, sans-serif;
+  font-size: 62.5%;
```

Comment:
It's better to font-size in px instead of percentages.

Reply:
Thanks, solved.